data/Makefile

```diff
@@ -1,16 +1,19 @@
 PYTHON ?= python3
 
-.PHONY: all clean
+.PHONY: all clean install_dependencies extract_requirements extract_types
 
 ECSS_DOC=ECSS-E-ST-10-06C6March2009.pdf
 REQUIREMENTS_TXT="ECSS-E-ST-10-06C-requirement-requirements.txt"
 TYPES_TXT="ECSS-E-ST-10-06C-requirement-types.txt"
 
-all: extract_requirements extract_types
+all: install_dependencies extract_requirements extract_types
 
 ${ECSS_DOC}:
 	wget https://ecss.nl/wp-content/uploads/standards/ecss-e/ECSS-E-ST-10-06C6March2009.pdf
 
+install_dependencies:
+	${PYTHON} -m pip install pdfplumber==0.11.6
```

Comment:
Shouldn't it be in a python requirement.txt file?

Reply:
I took a shortcut, but you are right, requirements.txt is nicer by convention, I will change that...